packages/manager/modules/manager-pci-common/src/api/data/instance.ts

```diff
@@ -1,6 +1,6 @@
 import { v6 } from '@ovh-ux/manager-core-api';
 
-export type TInstance = {
+export type TInstanceAPI = {
```

Comment:
i think we should keep TInstance type as API is implicit,
in your app you shoud TWorkflowInstance for example if you need to add custom fields